package.json

```diff
@@ -27,7 +27,6 @@
   "dependencies": {
     "@haventec/sjcl512": "1.0.0",
     "atob": "^2.1.2",
-    "clientjs": "^0.1.11",
     "fingerprintjs2": "^2.1.0"
   },
   "devDependencies": {
```

Comment:
Shall we delete also the @types/clientjs from the devDependencies?

Reply:
Removed - how did I miss that, thanks